src/pages/workspace/AccessOrNotFoundWrapper.tsx

```diff
@@ -155,7 +155,16 @@ function AccessOrNotFoundWrapper({
         if (pendingField && !isOffline && !isFeatureEnabled) {
             return;
         }
+        // If tax feature is disabled, for instance, from OD, we will navigate to the More Features page.
+        if (!isFeatureEnabled && featureName === CONST.POLICY.MORE_FEATURES.ARE_TAXES_ENABLED) {
```

Comment:
```suggestion
        if (Boolean(featureName) && !isFeatureEnabled) {
```

After looking at this more, let's actually implement the fix for all of the workspace feature pages. I see no reason why the behavior when disabling tax should be different from the behavior when disabling categories